### PR TITLE
deleting scaffold and create_adater_plugin.py file from scripts section of core as they a…

### PR DESCRIPTION
**Pull request by McKnight-42**
resolves #

### Description

deleting directory and a .py file from core as they are either deprecated or will live outside of core

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have added information about my change to be included in the [CHANGELOG](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry).

### CODE REVIEW

File: .changes/unreleased/Under the Hood-20220420-120025.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: removal of scaffold first attempt and create_adapter_plugin.py as they are deprecated
```
Comment (jtcohen6) [inline .changes/unreleased/Under the Hood-20220420-120025.yaml:2]: The changelog entry could be a good place to point to the new scaffold location: https://github.com/dbt-labs/dbt-database-adapter-scaffold

**Comment by leahwicz** [comment]
@McKnight-42 is this in draft waiting on other PRs to be reviewed first or are we ready to go here with this PR?

**Comment by McKnight-42** [comment]
@leahwicz I have it in draft cause we still have some Docs related PRs still up and i figured would be best to have everything as ready to go at once as we can

File: .changes/unreleased/Under the Hood-20220420-120025.yaml (hunk 1 above)
Comment (McKnight-42) [reply .changes/unreleased/Under the Hood-20220420-120025.yaml]: done! great idea!!!

**Comment by nathaniel-may** [comment]
@McKnight-42 do we want to delete entirely or leave the script with just a comment and print statement that points to the new repo?

**Comment by McKnight-42** [comment]
@nathaniel-may that's a good question, I know we were wanting to make it clear that we didn't want to use the .py file anymore and we link the new scaffold in the changelog, and will have it be whats referenced and pointed to in the docs. but if we feel changing out the file and still having it as a pointer would be worth it i'm down.

File: core/scripts/create_adapter_plugins.py [hunk 2]
```diff
@@ -1,323 +1,5 @@
...
-    url={url},
-    packages=find_namespace_packages(include=['dbt', 'dbt.*']),
-    include_package_data=True,
-    install_requires=[
-        "{dbt_core_str}",{dependencies}
-    ]
+# The create_adapter_plugins script is being replaced by a new interactive cookiecutter scaffold
+# that can be found https://github.com/dbt-labs/dbt-database-adapter-scaffold
+print(
+    "To create a new adapter please visit https://github.com/dbt-labs/dbt-database-adapter-scaffold"
```
Comment (nathaniel-may) [inline core/scripts/create_adapter_plugins.py:4]: Maybe add "This script has been deprecated. To create a new adapter..."

Comment (McKnight-42) [reply core/scripts/create_adapter_plugins.py]: Will do 👍

**Review by nathaniel-may — APPROVED** [review]

**Pull request merged**